src/Types/Interval.purs

```diff
@@ -508,7 +499,7 @@ maxSlot = wrap BigNum.maxValue
 data SlotToPosixTimeError
   = CannotFindSlotInEraSummaries Slot
   | StartingSlotGreaterThanSlot Slot
-  | EndTimeLessThanTime AbsTime
+  | EndTimeLessThanTime Number
```

Comment:
This error made me think about changing `Abstime` from `BiInt` to `Number` as most of the conversions are done by `Abstime` , but I don't know it would made sense in terms of `POSIXTime`.